common/src/sql/dbinit.sql

```diff
@@ -636,23 +636,22 @@ CREATE UNIQUE INDEX ON omicron.public.saga (
  * TODO more indexes for Saga?
  * - Debugging and/or reporting: saga_template_name? creator?
  */
-
 /*
- * TODO See notes above about cockroachdb/cockroach#57411 /
- * cockroachdb/cockroach#58084.
- */
--- CREATE TYPE omicron.public.SagaNodeEventType AS ENUM (
---     'started',
---     'succeeded',
---     'failed'
---     'undo_started'
---     'undo_finished'
+ * TODO: This is a data-carrying enum, see note on disk_state.
```

Comment:
Nit: link to https://github.com/oxidecomputer/omicron/issues/312?

Reply:
Done in f73fae5